src/components/MentorListDetails/MentorListDetails.jsx

```diff
@@ -1,68 +1,139 @@
+
+        return (
+            <>
+                <div className='mentor-detail-list'>
+                    <p id="mentor-detail-title">Find a Mentor</p>
+                    <div className='filter-group-mentor-details-dash'>
+                        <p>Onboarding Status</p>
+                        <select className='filter-mentor-details-dash' onChange={(e) => setSearchTermStatus(e.target.value)} value={searchTermStatus}>
+                            <option value="All">All</option>
+                            {status.map((currentValue, index) => (
+                                <option key={index} value={currentValue}>
+                                    {currentValue}
+                                </option>
+                            ))}
+                        </select>
+                        <p>Location</p>
+                        <select className='filter-mentor-details-dash' onChange={(e) => setSearchTermLocation(e.target.value)} value={searchTermLocation}>
+                            <option value="All">All</option>
+                            {locations.map((currentValue, index) => (
+                                <option key={index} value={currentValue}>
+                                    {currentValue}
+                                </option>
+                            ))}
+                        </select>
+                        <p>Skill</p>
+                        <select className='filter-mentor-details-dash' onChange={(e) => setSearchTermSkill(e.target.value)} value={searchTermSkill}>
+                            <option value="All">All</option>
+                            {skills.map((currentValue, index) => (
+                                <option key={index} value={currentValue}>
+                                    {currentValue}
+                                </option>
+                            ))}
+                        </select>
+                        <p>Rank</p>
+                        <select className='filter-mentor-details-dash' onChange={(e) => setSearchTermRank(e.target.value)} value={searchTermRank}>
+                            <option value="All">All</option>
+                            {ranks.map((currentValue, index) => (
+                                <option key={index} value={currentValue}>
+                                    {currentValue}
+                                </option>
+                            ))}
+                        </select>
+                    </div>
+
+                    {filteredMentors === undefined ? 
+                    <>
+                        <ul>No mentors</ul>
+                    </> :
+                        <ul>{filteredMentors.sort((a, b) => {
+                            return a.first_name - b.first_name
+                        }).map((mentorDataDetails, key) => {
+
+                            // TODO These should be a card component
```

Comment:
Agreed :)

Reply:
Might get to this, otherwise it's a post-preso task!

Reply:
Ended up doing it 😆